lib/oli_web/live/delivery/student_onboarding/wizard.ex

```diff
@@ -90,7 +90,8 @@ defmodule OliWeb.Delivery.StudentOnboarding.Wizard do
        steps: steps,
        current_step_name: @intro_step,
        current_step_index: 0,
-       datashop_session_id: datashop_session_id
+       datashop_session_id: datashop_session_id,
+       is_lti: !is_nil(session["lti_params_id"])
```

Comment:
`lti_params_id` is really intended to be used to facilitate the handshake and user creation after launch under the hood and may not always reliable to use.

I think a better method to determine if this is an LTI section is using the `section.open_and_free == false` (admittedly, this is a poor name for this flag since the introduction of independent sections that are not open and free) or `user.independent_learner == false` (see `lib/oli_web/components/delivery/utils.ex:72` is_independent_learner?).

I think `section.open_and_free == false` is probably best. We should eventually rename this to `section.independent`, but maybe at a later time.

Reply:
On further investigation, it appears we are using this to load and use `lti_params` in various other places, and it appears to be working fine so we can just continue with that approach.